src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,30 @@
+.catalog {
+  &__container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+
+    width: 944px;
+    margin: 0 auto;
+    padding: 50 auto;
+  }
+
+  &__card {
+    margin-right: $catalog-card-horizontal-right-margin;
+    margin-bottom: 46px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-child(-n + 4) {
+      margin-top: 50px;
```

Comment:
probably could have been without child, this style use for all your blocks

Reply:
Thx!
Done